### PR TITLE
Add a soft start module so the robot doesn't jerk to position dangerously

### PR DESCRIPTION
**Pull request by ysims**
The first thing the robot does generally is to run a zero walk command, causing it to jump very quickly to the standing position. This is a bit dangerous and does startle people who aren't expecting it. 

This PR adds in a module that takes control of the servos to begin with and moves the servos to the stand position over a few seconds at a low gain. The module will emit Done when it is happy, and as it is a root Task this will remove it from the tree. Unless it's requested again, it won't be active again, and as it's only in Startup reactions this won't happen.

Could be made a bit neater with scripts if the servos providers used position to determine done, but had some issues with it before.

### CODE REVIEW

File: module/strategy/StartSafely/src/StartSafely.cpp
Comment (JohanneMontano) [inline module/strategy/StartSafely/src/StartSafely.cpp:51]:
```
for (uint8_t i  = 0; i<config["servo_targets"].size(); ++i) {
    const auto& servo_name = config[i].first; // For the sake of clarity
    const auto& servo_position = config[i].second;
    cfg.servo_targets[i] = servo_position;
}
```

then servo_targets is just a sequence of map entries, or alternatively something that will make this code a lot shorter

**Review by TrentHouliston — COMMENTED** [review]
I would suggest just running the stand script instead.
That way the "default" position for the safe start will always be the same as any other stand in the system.

So even if you were not running a walk and instead were just doing stand this would still be safe

**Review by JohanneMontano — APPROVED** [review]
Changes have been addressed afaik, lgtm. Might be worth checking with @TrentHouliston  again just to be sure.

**Comment by Tom0Brien** [comment]
We should probably remove the soft start code in the `FootController` alongside this PR

**Comment by ysims** [comment]
Webots is running fine but I'll need to test again on a real robot as I was having issues last time with seg faulting on `keyboardwalk`.

File: roles/webots/demo.role
Comment (ysims) [inline roles/webots/demo.role:44]:
```suggestion
  strategy::StartSafely
```

**Comment by ysims** [comment]
This should be good to go, no issues when I was running it in webots last night or the real robot this morning.

**Review by JohanneMontano — APPROVED** [review]

**Pull request merged**
